zenodo/modules/deposit/static/templates/zenodo_deposit/actions.html

```diff
@@ -3,10 +3,10 @@
     <li class="pull-left">
       <button
         class="btn btn-danger"
+        data-toggle="modal"
+        data-target="#warningDeleteModal"
         ng-hide="recordsVM.invenioRecordsArgs.templateParams.is_published"
-        ng-disabled="recordsVM.invenioRecordsLoading || !recordsVM.invenioRecordsEndpoints.self"
-        ng-attr-title="{{(recordsVM.invenioRecordsLoading || !recordsVM.invenioRecordsEndpoints.self) && 'Record was not saved yet.' || 'Delete the deposit'}}"
-        ng-click="recordsVM.actionHandler(['self', 'DELETE'], recordsVM.invenioRecordsArgs.templateParams.is_new_version ? 'latest_html' : '/deposit')">
```

Comment:
removed the click part so it wouldn't perform the delete right away but rather show the popup

Reply:
Hm, just saw you also accidentally removed the `ng-attr-title` attribute, which provides the information mouse hover text. Can you add it back? (`ng-click` can stay removed of course)